api/base/filters.py

```diff
@@ -134,7 +134,7 @@ def _validate_operator(self, field, field_name, op):
                     detail="Field '{0}' does not support match operators in a filter.".format(field_name)
                 )
 
-    def _parse_date_param(self, field, field_name, op, value):
+    def _parse_date_param(self, field, source_field_name, op, value):
```

Comment:
Yes. In fact, that's how we got the value passed into this function in the first place...

I'm kind of on the fence about this one, personally. Ideally we'd only call convert key if we needed to resolve to the model source field... but in this case, parsing *always* calls it anyway.

Resolving the serializer--> model field name requires having the field object. If we called `convert_key` from other methods (like `query_params_to_odm_query`), we'd need to repeat the calls to `_get_field_or_error` that were already done when query params first got parsed.